src/components/navigation/AppNavigationBoard.vue

```diff
@@ -34,39 +34,65 @@
 			style="opacity: 0.5" />
 
 		<template v-if="!deleted" slot="actions">
-			<ActionButton v-if="canManage && !board.archived"
+			<ActionButton v-if="!showDueSettings"
+				icon="icon-more"
+				:close-after-click="true"
+				@click="actionDetails">
+				{{ t('deck', 'Board details') }}
+			</ActionButton>
+			<ActionButton v-if="canManage && !board.archived && !showDueSettings"
 				icon="icon-rename"
 				:close-after-click="true"
 				@click="actionEdit">
 				{{ t('deck', 'Edit board') }}
 			</ActionButton>
-			<ActionButton v-if="canManage && !board.archived"
+			<ActionButton v-if="canManage && !board.archived && !showDueSettings"
 				icon="icon-clone"
 				:close-after-click="true"
 				@click="actionClone">
 				{{ t('deck', 'Clone board ') }}
```

Comment:
btw trailing spaces inside the translation?

Reply:
Thanks, pushed a fix here to avoid conflicts, even though not very related, since it is a very minor change.